src/index.js

```diff
@@ -1,2 +1,65 @@
-export function createFetchAction() {}
-export function createFetchReducer() {}
+import { CALL_API } from 'redux-api-middleware';
+
+import { createPendingType, createDefaultBailout, overrideOptions } from './util-action';
+import { fetchSuccess, fetchFailure, fetchPending } from './util-reducer';
+
+/**
+  * creates action for data fetching
+  * @param {string} [id] - identifier for the action
+  * @param {string} [url] - url to fetch
+  * @param {object} [options] - options to consider when creating action
+  * @returns {object} - redux-api-middleware compatible action
+  */
+export function createFetchAction(id, url, options = { force: false, method: 'GET', body: '' }) {
```

Comment:
`id` is called `name` in the README.md

Also I think it should be clearer in someway that the `id`/`name` should be identical for it to work. Maybe recommend that a constant is used?